docs/aiops-projects.md

```diff
@@ -0,0 +1,41 @@
+# AI Ops Projects 
```

Comment:
Can we order the projects alphabetically? Or do they currently have a specific order?

Reply:
No specific order as of now, I'll order them alphabetically